frontend/cypress/integration/common/overview.ts

```diff
@@ -0,0 +1,135 @@
+import { And, Before, Given, Then } from "cypress-cucumber-preprocessor/steps";
+
+const url = "/console";
+
+Before(() => {
+    // Focing to not stop cypress on unexpected errors not related to the tests.
+    // There are some random failures due timeouts/loadtime/framework that throws some error in the browser.
+    // After reviewing the tests failures, those are unrelated to the app, so,
+    // it needs this event to not fail the CI action due some "slow" action or similar.
+    // This is something to review in future iterations when tests are solid, but I haven't found a better way to
+    // solve this issue.
+    cy.on("uncaught:exception", (err, runnable) => {
+        return false;
+    });
+});
+
+Given('user opens the overview page', () => {
+    // Forcing "Pause" to not cause unhandled promises from the browser when cypress is testing
```

Comment:
:+1: I've noticed these error messages before about unhandled promises. Perhaps the frontend needs to more gracefully handle request cancellations when navigating away from a page with in-flight requests?

Reply:
Well, it's a combination, in theory, the "in-flight" requests are properly canceled, but this was more noise that the test takes more time than the default refresh poll. Then it creates noise and makes the test longer.